website/src/documentation/io/built-in-google-bigquery.md

```diff
@@ -167,11 +167,15 @@ schema](#creating-a-table-schema) covers schemas in more detail.
 ## Reading from BigQuery
 
 BigQueryIO allows you to read from a BigQuery table, or read the results of
-an arbitrary SQL query string. When you apply a BigQueryIO read transform,
-Beam invokes a [BigQuery export request](https://cloud.google.com/bigquery/docs/exporting-data).
-Beam’s use of this API is subject to BigQuery's [Quota](https://cloud.google.com/bigquery/quota-policy#export)
-and [Pricing](https://cloud.google.com/bigquery/pricing) policies.
+an arbitrary SQL query string. By default, when you apply a BigQueryIO read
+transform, Beam invokes a [BigQuery export request](https://cloud.google.com/bigquery/docs/exporting-data).
+However, the Beam SDK for Java (version 2.11.0 and later) supports using the
+[BigQuery Storage API](#storage-api) when reading from a table. Using the
+BigQuery Storage API with a query string is not supported.
```

Comment:
Nit: this should probably be "not currently supported", since it's definitely in-plan to support reading from query results using a destination table with the storage API.

Reply:
Done